db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
There's a lot of different options here.  Do you have some mechanism for catching drift?